stanford-earth/SE3_BLT
# EARTH-000: Re-sync with features an prod.

# READY FOR REVIEW

# Summary
- Re-syncing composer versions and feature exports

# Needed By (Date)
- EOW

# Steps to Test

1. Check out this branch
2. Run composer update 
3. run `blt sync:refresh`
4. Run `drush cex`
5. Validate no new changes.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
